forecasting/src/autogluon/forecasting/__init__.py

```diff
@@ -1,10 +1,21 @@
 import logging
+from packaging.version import parse
 
 try:
     from .version import __version__
 except ImportError:
     pass
 
+try:
+    from mxnet import __version__ as mxnet_version
+
+    assert parse("2.0") > parse(mxnet_version) >= parse("1.7")
+except (ImportError, AssertionError):
+    raise ImportError(
+        "autogluon.forecasting depends on Apache MxNet v1.7 or greater (below v2.0). "
+        "Please install a suitable version of MxNet in order to use autogluon.forecasting."
+    )
```

Comment:
Unclear how to install mxnet from this instruction (Ditto for warning in setup.py). Either provide a pip install command or a link to a website with more info.